frontend/src/components/Body/BodyOne/BodyOne.js

```diff
@@ -0,0 +1,45 @@
+// LOGO
+// IMG / BANNER
+// Description & image
```

Comment:
i assume these comments were for your own notes? if so, let's remove here and in other files!